From 7fa48b42661af81bb30fcb57e7fd2bb31cf9739d Mon Sep 17 00:00:00 2001
From: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Date: Fri, 12 Sep 2014 23:12:29 +0300
Subject: [PATCH 287/456] v4l: xilinx: cfa: Rename compatible string to
 xlnx,v-cfa

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

This matches the compatible string values used by the DT generator.

Signed-off-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
(cherry picked from commit a620c6b449161a502c280a8ed7741d142bf42cfd)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 .../bindings/media/xilinx/xlnx,axi-cfa.txt         |   52 --------------------
 .../bindings/media/xilinx/xlnx,v-cfa.txt           |   52 ++++++++++++++++++++
 drivers/media/platform/xilinx/xilinx-cfa.c         |    2 +-
 3 files changed, 53 insertions(+), 53 deletions(-)
 delete mode 100644 Documentation/devicetree/bindings/media/xilinx/xlnx,axi-cfa.txt
 create mode 100644 Documentation/devicetree/bindings/media/xilinx/xlnx,v-cfa.txt

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,axi-cfa.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,axi-cfa.txt
deleted file mode 100644
index dac15fb..0000000
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,axi-cfa.txt
+++ /dev/null
@@ -1,52 +0,0 @@
-Xilinx Color Filter Array (CFA)
--------------------------------
-
-Required properties:
-
-- compatible: Must be "xlnx,axi-cfa-7.0".
-
-- reg: Physical base address and length of the registers set for the device.
-
-- ports: Video ports, using the DT bindings defined in ../video-interfaces.txt.
-  The cfa has an input port (0) and an output port (1).
-
-Required port properties:
-
-- xlnx,axi-video-format: Must be one of "rggb", "grbg", "gbrg", and "bggr" for
-  the input port (0), and "rbg" for the output port (1).
-
-- xlnx,axi-video-width: Video width as defined in video.txt
-
-Example:
-
-	axi_cfa_0: axi_cfa {
-		compatible = "xlnx,axi-cfa-7.0";
-		reg = <0x400b0000 0x10000>;
-
-
-		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
-			port@0 {
-				reg = <0>;
-
-				xlnx,axi-video-format = "rggb";
-				xlnx,axi-video-width = <8>;
-
-				cfa0_in: endpoint {
-					remote-endpoint = <&spc0_out>;
-				};
-			};
-			port@1 {
-				reg = <1>;
-
-				xlnx,axi-video-format = "rbg";
-				xlnx,axi-video-width = <8>;
-
-				cfa0_out: endpoint {
-					remote-endpoint = <&ccm0_in>;
-				};
-			};
-		};
-	};
diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-cfa.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-cfa.txt
new file mode 100644
index 0000000..653bfca
--- /dev/null
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-cfa.txt
@@ -0,0 +1,52 @@
+Xilinx Color Filter Array (CFA)
+-------------------------------
+
+Required properties:
+
+- compatible: Must be "xlnx,v-cfa-7.0".
+
+- reg: Physical base address and length of the registers set for the device.
+
+- ports: Video ports, using the DT bindings defined in ../video-interfaces.txt.
+  The cfa has an input port (0) and an output port (1).
+
+Required port properties:
+
+- xlnx,axi-video-format: Must be one of "rggb", "grbg", "gbrg", and "bggr" for
+  the input port (0), and "rbg" for the output port (1).
+
+- xlnx,axi-video-width: Video width as defined in video.txt
+
+Example:
+
+	axi_cfa_0: axi_cfa {
+		compatible = "xlnx,v-cfa-7.0";
+		reg = <0x400b0000 0x10000>;
+
+
+		ports {
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			port@0 {
+				reg = <0>;
+
+				xlnx,axi-video-format = "rggb";
+				xlnx,axi-video-width = <8>;
+
+				cfa0_in: endpoint {
+					remote-endpoint = <&spc0_out>;
+				};
+			};
+			port@1 {
+				reg = <1>;
+
+				xlnx,axi-video-format = "rbg";
+				xlnx,axi-video-width = <8>;
+
+				cfa0_out: endpoint {
+					remote-endpoint = <&ccm0_in>;
+				};
+			};
+		};
+	};
diff --git a/drivers/media/platform/xilinx/xilinx-cfa.c b/drivers/media/platform/xilinx/xilinx-cfa.c
index 77853d4..31f6072 100644
--- a/drivers/media/platform/xilinx/xilinx-cfa.c
+++ b/drivers/media/platform/xilinx/xilinx-cfa.c
@@ -370,7 +370,7 @@ static int xcfa_remove(struct platform_device *pdev)
 static SIMPLE_DEV_PM_OPS(xcfa_pm_ops, xcfa_pm_suspend, xcfa_pm_resume);
 
 static const struct of_device_id xcfa_of_id_table[] = {
-	{ .compatible = "xlnx,axi-cfa-7.0" },
+	{ .compatible = "xlnx,v-cfa-7.0" },
 	{ }
 };
 MODULE_DEVICE_TABLE(of, xcfa_of_id_table);
-- 
1.7.5.4

